modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main_view/spaces/SpaceSummaryHeader.tsx

```diff
@@ -88,16 +88,16 @@ export default function SpaceSummaryHeader({
 		<div className="align-items-center d-flex justify-content-between">
 			<h2 className="font-weight-semi-bold m-0 text-4">{title}</h2>
 
-			{spaceModalProps ? (
+			{url ? (
+				<ClayLink href={url}>{label}</ClayLink>
```

Comment:
```suggestion
			<ClayLink className="text-3 text-weight-semi-bold" href={url}>
				{label}
			</ClayLink>
```

Could you apply these classes in `ClayLink`?
I made this but probably was lost in some rebase.

https://github.com/brianchandotcom/liferay-portal/commit/65ba7c98745b36e27d0224b35481fd27a185df6c